src/IpInterface.php

```diff
@@ -114,44 +114,71 @@ public function isCompatible();
     public function isEmbedded();
 
     /**
-     * Whether the IP is reserved for link-local usage according to
+     * Whether the IP is reserved for link-local usage, according to
      * RFC 3927/RFC 4291 (IPv4/IPv6).
      *
      * @return bool
      */
     public function isLinkLocal();
 
     /**
-     * Whether the IP is a loopback address according to RFC 2373/RFC 3330
+     * Whether the IP is a loopback address, according to RFC 2373/RFC 3330
      * (IPv4/IPv6).
      *
      * @return bool
      */
     public function isLoopback();
 
     /**
-     * Whether the IP is a multicast address according to RFC 3171/RFC 2373
+     * Whether the IP is a multicast address, according to RFC 3171/RFC 2373
      * (IPv4/IPv6).
      *
      * @return bool
      */
     public function isMulticast();
 
     /**
-     * Whether the IP is for private use according to RFC 1918/RFC 4193
+     * Whether the IP is for private use, according to RFC 1918/RFC 4193
      * (IPv4/IPv6).
      *
      * @return bool
      */
     public function isPrivateUse();
 
     /**
-     * Whether the IP is unspecified according to RFC 5735/RFC 2373 (IPv4/IPv6).
+     * Whether the IP is unspecified, according to RFC 5735/RFC 2373 (IPv4/IPv6).
      *
      * @return bool
      */
     public function isUnspecified();
 
+    /**
+     * Whether the IP is reserved for network devices benchmarking, according
+     * to RFC 2544/RFC 5180 (IPv4/IPv6).
+     *
+     * @return bool
+     */
+    public function isBenchmarking();
+
+    /**
+     * Whether the IP is in range designated for documentation, according to
+     * RFC 5737/RFC 3849 (IPv4/IPv6).
+     *
+     * @return bool
+     */
+    public function isDocumentation();
+
+    /**
+     * Whether the IP appears to be publicly/globally routable. Please refer to
+     * the IANA Special-Purpose Address Registry documents.
+     *
+     * @see https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml
+     * @see https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv6-special-registry.xhtml
+     *
+     * @return bool
+     */
+    public function isPublic();
```

Comment:
Change to `isPublicUse()` to reflect same naming convention as `isPrivateUse()`?